.github/workflows/run-archiver.yml

```diff
@@ -6,7 +6,7 @@ on:
     inputs:
       datasets:
         description: 'Comma-separated list of datasets to archive (e.g., "ferc2","ferc6").'
-        default: '"eia176","eia191","eia757a","eia860","eia860m","eia861","eia923","eia930","eiaaeo","eiawater","eia_bulk_elec","epacamd_eia","epacems",ferc1","ferc2","ferc6","ferc60","ferc714","mshamines","nrelatb","phmsagas","vcerare"'
+        default: '"eia176","eia191","eia757a","eia860","eia860m","eia861","eia923","eia930","eiaaeo","eiawater","eia_bulk_elec","epacamd_eia","epacems","ferc1","ferc2","ferc6","ferc60","ferc714","mshamines","nrelatb","phmsagas","vcerare"'
```

Comment:
Not sure how I did it, but this one character error was my fault. Introduced when I switched us off the bigger paid runner.

Reply:
And it doesn't get caught by the action linter because it's an error in our little JSON blob.

Reply:
Ugh. Slipped past both of us. Well, at least you caught it now!